Apeswap-Bots/APESWAP-BOT-2/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "apeswap-2",
+  "version": "0.0.1",
+  "description": "detects new address inclusion and exclusion in the golden banana token",
+  "chainIds": [
+    56
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest",
+    "format": "prettier --write \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "forta-agent": "^0.1.3"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
+    "jest": "^27.0.6",
+    "nodemon": "^2.0.8",
+    "prettier": "^2.6.0",
+    "ts-jest": "^27.0.3",
+    "typescript": "^4.3.4",
+    "forta-agent-tools": "^2.0.5"
+  },
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/%40apeswap/AS2-address-inclusion-exclusion/Apeswap-Bots"
```

```suggestion
  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/Apeswap-Bots/APESWAP-BOT-2"
```
Let's use this address as it is going to be the final address of the bot